### PR TITLE
Stop effect in more-info-light

### LINKED ISSUE #3619: Add Stop Effect to Lights more-info dialog
**Description of problem:**
You can turn on light effects from the more-info dialog for Lights, but there is no apparent way to turn these effects off. Seems that you have to turn the light off and then back on again, which is not obvious and can be a little confusing. Some examples of community posts referencing this: [here](https://community.home-assistant.io/t/hue-light-effects-wont-stop/104340), [here](https://community.home-assistant.io/t/stopping-hue-colorloop-without-turning-off-lights/55083), [here](https://community.home-assistant.io/t/turn-off-colorloop/58657).

It seems that sending `effect: stop` will stop the effect and restore your lights. I've tested this with Hue and it seems to work, but I am not 100% positive if this is a standard Home Assistant thing that all light platforms support? 

I think that there should either be a "stop" option added inside of the dropdown when an effect is currently active like this (mockup 1):

stopeffectmock1

Or a stop button to the right of the "Effect" dropdown as pictured here (mockup 2):

stopeffectmock2

### PR DESCRIPTION
**Pull request by iantrich**
Closes https://github.com/home-assistant/home-assistant-polymer/issues/3619

### CODE REVIEW

**Comment by balloob** [comment]
Why did you go for a button? Since the dropdown represents current effect, having "none" as an entry in there seems to make sense.

**Comment by iantrich** [comment]
It felt cleaner to use a button rather than messing with the effect_list and made it more apparent

**Comment by balloob** [comment]
It wouldn't be a lot of messing, you can just write a `<paper-item>None</paper-item>` above the repeat template

**Comment by iantrich** [comment]
Ah cool, didn't think of that. I feel so out of my element in Polymer 😄

**Comment by balloob** [comment]
Yeah let's killllll it 😄 

(btw, it would be the same in Lit, but then put above the `map`)

File: src/dialogs/more-info/controls/more-info-light.js [hunk 1]
```diff
@@ -37,6 +38,19 @@ class MoreInfoLight extends LocalizeMixin(EventsMixin(PolymerElement)) {
...
+        ha-paper-dropdown-menu {
+          max-width: 150px;
+        }
+
+        .effect_list {
+          display: flex;
+          align-items: flex-end;
+        }
+
+        .stop {
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-light.js]: Not used anymore?

File: src/dialogs/more-info/controls/more-info-light.js [hunk 2]
```diff
@@ -188,6 +189,7 @@ class MoreInfoLight extends LocalizeMixin(EventsMixin(PolymerElement)) {
               on-selected-changed="effectChanged"
               attr-for-selected="item-name"
             >
+              <paper-item item-name$="stop">stop</paper-item>
```
Comment (balloob) [inline src/dialogs/more-info/controls/more-info-light.js]: Is `stop` handled accordingly? Shouldn't we set effect to `none` to stop it ?

**Comment by SeanPM5** [comment]
I was going off the solution in [this forum post](https://community.home-assistant.io/t/turn-off-colorloop/58657/4). It seems to work for me on Hue?

This is how I tested:
1) Start `colorloop` effect from lights more-info
2) Go to Services tab, select `light.turn_on` and paste into service data:
```
{
  "entity_id": "light.floor_lamp",
  "effect": "stop"
}
```

It stopped the `colorloop` effect and restored the lights to what they were. effect `none` also worked the same way. So it seems like at least the Hue platform supports both methods?

**Comment by balloob** [comment]
Hue just matches any unknown effect as show no effect. That is Hue specific. 

We should set effect to `null` and send that.

**Comment by iantrich** [comment]
I'll test that tonight. I do have an old yeelight actually in a drawer that should support effects that I can test on. Forgot about that thing

**Comment by iantrich** [comment]
The only thing that works for my yeelight is

```json
effect: "Stop"
```
Lowercase "stop", "none" or "null" cause an error:
```
connection-mixin.ts:60 Error calling service light turn_on {entity_id: "light.girl_s_light", effect: "stop"} {code: "unknown_error", message: "local variable 'flow' referenced before assignment"}
```

Can you try "Stop" @SeanPM5?

**Comment by SeanPM5** [comment]
"Stop" works fine without errors on Hue. Based on what balloob said earlier it will interpret anything that is not "colorloop" or "random" as meaning stop. So Hue isn't picky.

Maybe @dshokouhi can test "Stop" effect on his Lightify and Limitlessled? It would be nice if we could find something that works on everything.

**Comment by dshokouhi** [comment]
my lightify lights do not support colorloop and random just changes the color once.  Limitlessled does not stop if i send "stop" "Stop" or "none" I have to change the color of the light manually to get it to stop.  I think this might be platform specific?

**Comment by iantrich** [comment]
I think I'm going to revert the change for now

**Comment by SeanPM5** [comment]
Yeah, bummer, it seems there isn't much consistency in the various light platforms unfortunately ☹️

**Pull request merged**